### ISSUE TITLE
iTunes Tags

### ISSUE THREAD

**Issue by tboneonair** [amsehili/genRSS#24]
Is it possible to add the following iTunes tags:

-  <itunes:duration>300</itunes:duration>
-  <itunes:title>The Example Times Daily News Briefing</itunes:title>
-  <itunes:block>yes</itunes:block>

And copyright tags: <copyright>Copyright 2018 The Example Times – For Personal Use Only</copyright>

Everything about this script is great. I'm hoping to generate an RSS feed for Google that requires the following:

https://developers.google.com/news/assistant/newsbriefings/technical-requirements

Any help would be greatly appreciated.

**Comment by KAMiKAZOW** [comment]
For professional podcast publishing you may want to have a look at something like https://publisher.podlove.org/

That said, iTunes tags would be great to have.

**Comment by tboneonair** [comment]
> For professional podcast publishing you may want to have a look at something like https://publisher.podlove.org/
> 
> That said, iTunes tags would be great to have.

I agree, and like what Podlove is offering, but I am hoping to automate the process. 

Audio is placed into a directory, and every 30 min, I have genRSS set to run and generate my feed.

**Comment by tboneonair** [comment]
I also see references to itunes:duration but have no idea how to use it. 

`          <item>
             <guid>item1</guid>
             <link>my/web/site/media/item1</link>
             <title>Title of item 1</title>
             <description>This is item 1</description>
             <pubDate>Mon, 22 Dec 2014 18:30:00 +0000</pubDate>
          </item>

    >>> item = buildItem("my/web/site/media/item2", title = "Title of item 2", indent = " ",
    ...                  extraTags=[{"name" : "itunes:duration" , "value" : "06:08"}])
    >>> print(item)
      <item>
       <guid>my/web/site/media/item2</guid>
       <link>my/web/site/media/item2</link>
       <title>Title of item 2</title>
       <description></description>
       <itunes:duration>06:08</itunes:duration>
      </item>
`

**Comment by amsehili** [comment]
`<itunes:duration>` tag has been added, other itunes tag should be easier to add. Will fix this as soon as I can, please feel free to contribute.
